WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -373,38 +376,40 @@ extension GutenbergViewController: GutenbergBridgeDelegate {
         return WPMediaType.all
     }
 
-    func gutenbergDidRequestMediaFromSiteMediaLibrary(filter: WPMediaType, with callback: @escaping MediaPickerDidPickMediaCallback) {
+    func gutenbergDidRequestMediaFromSiteMediaLibrary(filter: WPMediaType, allowMultipleSelection: Bool, with callback: @escaping MediaPickerDidPickMediaCallback) {
         mediaPickerHelper.presentMediaPickerFullScreen(animated: true,
                                                        filter: filter,
                                                        dataSourceType: .mediaLibrary,
+                                                       allowMultipleSelection: allowMultipleSelection,
                                                        callback: {(asset) in
-                                                        guard let media = asset as? Media else {
-                                                            callback(nil, nil)
+                                                        guard let media = asset as? [Media] else {
+                                                            callback(nil)
                                                             return
                                                         }
                                                         self.mediaInserterHelper.insertFromSiteMediaLibrary(media: media, callback: callback)
         })
     }
 
-    func gutenbergDidRequestMediaFromDevicePicker(filter: WPMediaType, with callback: @escaping MediaPickerDidPickMediaCallback) {
+    func gutenbergDidRequestMediaFromDevicePicker(filter: WPMediaType, allowMultipleSelection: Bool, with callback: @escaping MediaPickerDidPickMediaCallback) {
         mediaPickerHelper.presentMediaPickerFullScreen(animated: true,
                                                        filter: filter,
                                                        dataSourceType: .device,
+                                                       allowMultipleSelection: allowMultipleSelection,
                                                        callback: {(asset) in
-                                                        guard let phAsset = asset as? PHAsset else {
-                                                            callback(nil, nil)
+                                                        guard let phAsset = asset as? [PHAsset] else {
```

Comment:
I know this is still a draft but just wanted to point out the need for some variable names updated as plural.